player/playloop.c

```diff
@@ -619,32 +618,25 @@ static void handle_pause_on_low_cache(struct MPContext *mpctx)
 
     if (mpctx->restart_complete && use_pause_on_low_cache) {
         if (mpctx->paused && mpctx->paused_for_cache) {
-            if (!s.underrun && (!opts->cache_pausing || s.idle ||
-                                s.ts_duration >= mpctx->cache_wait_time))
+            if (!s.underrun && (!opts->cache_pause || s.idle ||
+                                s.ts_duration >= opts->cache_pause_wait))
             {
-                double elapsed_time = now - mpctx->cache_stop_time;
-                if (elapsed_time > mpctx->cache_wait_time) {
-                    mpctx->cache_wait_time *= 1.5 + 0.1;
```

Comment:
Very happy to see this go away. It was definitely responsible for a lot of "bullshit" behavior.